packages/framework/ai-collab/src/explicit-strategy/index.ts

```diff
@@ -58,30 +65,14 @@ export interface GenerateTreeEditsOptions {
 	planningStep?: boolean;
 }
 
-interface GenerateTreeEditsSuccessResponse {
-	status: "success";
-	tokensUsed: TokenUsage;
-}
-
-interface GenerateTreeEditsErrorResponse {
-	status: "failure" | "partial-failure";
-	errorMessage: "tokenLimitExceeded" | "tooManyErrors" | "tooManyModelCalls" | "aborted";
-	tokensUsed: TokenUsage;
-}
-
 /**
- * Prompts the provided LLM client to generate valid tree edits.
- * Applies those edits to the provided tree branch before returning.
- *
- * @remarks
- * - Optional root nodes are not supported
- * - Primitive root nodes are not supported
- *
- * @internal
+ * The editLog is transformed into an array of Diff objects. Each Diff object includes
```

Comment:
"editLog" is something that's defined within the function - the caller of this function likely won't know what it is. We should express the docs here in terms of what the function means to the caller.